src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -801,28 +809,6 @@ private void PurgeOldLogDirectories()
             }
         }
 
-        public static ScriptHost Create(IScriptHostEnvironment environment, IScriptEventManager eventManager,
```

Comment:
Removing this factory method in favor of separating out construction and initialization. I needed this ability to be able to subscribe to the new event before Initialize is called.